program/src/main/QuestionSet.vala

```diff
@@ -38,8 +48,21 @@ public class QuestionSet : Object{
         return rubricPool.get(ID);
     }
 
-    public void resizeBounds(double[] d){
-        
+    //There's probably a better way to do this but I'm new to vala
+    public bool bounds_is_null() { 
```

Comment:
If this comment is for reviewers, it's best to leave it in the PR and not in the code. Also a better name would be 'are_bounds_valid'

Reply:
Changed this to are_bounds_valid - turns out it's only ever called once so I guess it technically doesn't HAVE to be a function but it might be nice to have for the future/just as a general function for this class to have lol